### DISCUSSION TITLE
Where do I find version differences?

### DISCUSSION THREAD

**Discussion by BrunoZockt** [cc-tweaked/CC-Tweaked#1372] [Q&A] [answered]
CC:Tweaked is available for most major minecraft versions from 1.7.10 upwards. I'm assuming that not all the versions are being actively updated and maintained. I want to write a computer program that runs on any CC:Tweaked version which means I need to know the differences between the versions. Is there documentation on this somewhere?

**Comment by SquidDev** [comment, marked as answer]
There's a couple of places you can check:
 - [The documentation site](https://tweaked.cc/) mentions in which versions methods were introduced or majorly changed. See for instance [`term.blit`](https://tweaked.cc/module/term.html#v:blit), which mentions it was introduced in 1.74 (so is available on MC 1.7.10) but then extended in 1.80pr1 (which is 1.12+ only).
 - [The changelog](https://github.com/cc-tweaked/CC-Tweaked/blob/mc-1.19.x/projects/core/src/main/resources/data/computercraft/lua/rom/help/changelog.md) contains a more complete list of everything which changed between versions.
